worlds/undertale/Regions.py

```diff
@@ -0,0 +1,99 @@
+import random
```

Comment:
Do not use random from the random module. Instead use the `MultiWorld.random` property. Otherwise, two games generated with the same seed will result in different outputs.